.storybook/preview.ts

```diff
@@ -29,3 +29,31 @@ export const parameters = {
     },
   },
 };
+
+export const globalTypes = {
+  version: {
+    name: 'version',
+    description: 'Select version Stable/Beta',
+    toolbar: {
+      icon: 'branch',
+      items: [
+        { value: 'stable', icon: 'branch', title: 'Stable Version' },
+        { value: 'beta', icon: 'branch', title: 'Beta(next) Version' },
```

Comment:
We don't need to include 'Next' in parentheses; 'Beta Version' is sufficient.

Reply:
ok, i changed